x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -20,37 +20,79 @@ export const ManualInstructions: React.FunctionComponent<Props> = ({
   apiKey,
   kibanaCASha256,
 }) => {
-  const command = `
-./elastic-agent enroll ${kibanaUrl} ${apiKey.api_key}${
+  const enrollArgs = `${kibanaUrl} ${apiKey.api_key}${
     kibanaCASha256 ? ` --ca_sha256=${kibanaCASha256}` : ''
-  }
+  }`;
+  const macOsLinuxTarCommand = `
+./elastic-agent enroll ${enrollArgs}
 ./elastic-agent run`;
+
+  const linuxDebRpmCommand = `
+./elastic-agent enroll ${enrollArgs}
+systemctl enable elastic-agent
+systemctl start elastic-agent`;
+
+  const windowsCommand = `
+./elastic-agent enroll ${enrollArgs}
+./install-service-elastic-agent.ps1`;
+
   return (
     <>
       <EuiText>
         <FormattedMessage
           id="xpack.ingestManager.enrollmentInstructions.descriptionText"
-          defaultMessage="From the agent’s directory, run these commands to enroll and start the Elastic Agent. {enrollCommand} will write to your agent’s configuration file so that it has the correct settings. You can use this command to setup agents on more than one host."
+          defaultMessage="From the agent’s directory, run the appropriate commands to enroll and start an Elastic Agent. You can reuse these commands to setup agents on more than one machine."
+        />
+      </EuiText>
+      <EuiSpacer size="l" />
+      <EuiTitle size="xs">
+        <h4>
+          <FormattedMessage
+            id="xpack.ingestManager.enrollmentInstructions.windowsTitle"
+            defaultMessage="Windows"
+          />
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiCodeBlock fontSize="m" isCopyable={true} paddingSize="m">
+        <pre style={{ overflow: 'scroll' }}>{windowsCommand}</pre>
+      </EuiCodeBlock>
+      <EuiSpacer size="l" />
+      <EuiTitle size="xs">
+        <h4>
+          <FormattedMessage
+            id="xpack.ingestManager.enrollmentInstructions.linuxDebRpmTitle"
+            defaultMessage="Linux (.deb and .rpm)"
+          />
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiCodeBlock fontSize="m" isCopyable={true} paddingSize="m">
+        <pre style={{ overflow: 'scroll' }}>{linuxDebRpmCommand}</pre>
+      </EuiCodeBlock>
+      <EuiSpacer size="l" />
+      <EuiTitle size="xs">
+        <h4>
+          <FormattedMessage
+            id="xpack.ingestManager.enrollmentInstructions.macLinuxTarTitle"
+            defaultMessage="macOS / Linux (.tar.gz)"
+          />
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiCodeBlock fontSize="m" isCopyable={true} paddingSize="m">
+        <pre style={{ overflow: 'scroll' }}>{macOsLinuxTarCommand}</pre>
+      </EuiCodeBlock>
+      <EuiSpacer size="l" />
+      <EuiText>
+        <FormattedMessage
+          id="xpack.ingestManager.enrollmentInstructions.macLinuxTarInstructions"
+          defaultMessage="You will need to run {command} if the agent’s system reboots. This is a known limitiation in 7.9."
```

Comment:
I feel that we don't need to call the release version here. @dedemorton can you suggest better text for this?

Reply:
@jen-huang @dedemorton I think it is very helpful to have *some message here about persistence support. However if we wish to change it to say 'current release' instead of '7.9' I won't argue it, except to say it would be good to get this in all the sooner to get the back-port done.

Reply:
@jen-huang Looks like you worked this out? I think it's best to describe what users need to do rather than calling this out as a known limitation in this version.